docs/embedding/sdk/config.md

```diff
@@ -13,7 +13,7 @@ To use the SDK in your app, you need to import the `MetabaseProvider` component
 ```typescript
 const config = defineEmbeddingSdkConfig({
   metabaseInstanceUrl: "https://metabase.example.com", // Required: Your Metabase instance URL
-  authProviderUri: "https://app.example.com/sso/metabase", // Required: An endpoint in your app that returns signs the user in and delivers a token
+  authProviderUri: "https://app.example.com/sso/metabase", // Required: An endpoint in your app that signs the user in and delivers a token
```

Comment:
```suggestion
  authProviderUri: "https://app.example.com/sso/metabase", // Required: An endpoint in your app that signs the user in and returns a session
```
The endpoint signs the jwt token, sends it and returns a session. I'm not sure what "delivers a token" was originally meant to mean I find it confusing

Reply:
That's a good point.

Reply:
Fixed in 2aec4fada90030dc63b95491bf76764fabcba18e